modules/command-engine/javadsl/src/main/scala/surge/javadsl/command/AggregateRef.scala

```diff
@@ -18,16 +19,13 @@ trait AggregateRef[Agg, Cmd, Event] {
   def applyEvent(event: Event): CompletionStage[ApplyEventResult[Agg]]
 }
 
-final class AggregateRefImpl[AggId, Agg, Cmd, Event](val aggregateId: AggId, val region: ActorRef, val tracer: Tracer)
+final class AggregateRefImpl[AggId, Agg, Cmd, Event](val aggregateId: AggId, protected val region: ActorRef, protected val tracer: Tracer)
     extends AggregateRef[Agg, Cmd, Event]
+    with AggregateRefBaseTrait[AggId, Agg, Cmd, Event]
     with AggregateRefTrait[AggId, Agg, Cmd, Event] {
```

Comment:
Really minor but this probably only needs to extend `AggregateRefBaseTrait` since that's already pulling in the core `AggregateRefTrait`

Reply:
Not sure what you mean by "core `AggregateRef`"

`AggregateRef` here is `surge.javadsl.AggregateRef` defined in